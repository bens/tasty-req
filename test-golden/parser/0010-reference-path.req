1: POST /
{"a": 0}
---
{:r 1!} <> {"b": {:r 1!["a"]}}
