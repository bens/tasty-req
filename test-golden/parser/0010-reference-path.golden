[ Command
    { command'ident = 1
    , command'always = False
    , command'method = "POST"
    , command'url = [ Right "/" ]
    , command'request_body =
        [ JsonObject (fromList [ ( "a" , JsonNumber 0.0 ) ]) ]
    , command'response_body =
        [ JsonCustom "r" (Left (Ref 1 Response []))
        , JsonObject
            (fromList
               [ ( "b" , JsonCustom "r" (Left (Ref 1 Response [ "a" ])) ) ])
        ]
    }
]
